README.md

```diff
@@ -9,3 +9,24 @@ The media player will search for songs in the `/home/{user}/Music/SongLibrary` f
 ## Class Diagram
 
 ![Class Diagram](pictures/ClassDiagram.png)
+
+## Download Songs
+If you only need a small number of song, you can run the `./DownloadSongs.sh` in the terminal to download a pre-selected selection of songs.
+
+To download all the songs hosted on the AWS bucket:
+
+**1. Install AWS CLI**
+
+In order to download all song, you need to install AWS CLI by running `sudo apt-get install awscli`
+
+**2. Obtain Access Keys**
+
+To directly download from the aws bucket, you will need to obtain the Access Key ID and the Secret Access Key.
+
+**3. Configure the AWS CLI**
+
+Run `aws configure` in the terminal to configure the profile for AWS. Input the access keys obtained. The region name is `us-west-2`.
```

Comment:
Seperate sentences so it's one line each, it's easier to read and still shows up in the rich text view as one paragraph.
```
Run `aws configure` in the terminal to configure the profile for AWS.
Input the access keys obtained.
The region name is `us-west-2`.
```